lib/turn.rb

```diff
@@ -0,0 +1,24 @@
+class Turn
+  attr_reader :guess, :card
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if guess == card.answer
+      true
+    else
+      false
+    end
+  end
```

Comment:
you can simplify this method to just one line: 


```
  def correct?
     guess == card.answer
  end
```

since == will either return true or false automatically.